forum/qa-content/qa-page.js

```diff
@@ -568,8 +563,19 @@ function qa_ajax_error()
 			copyCodeBtn.textContent = 'Kopiuj';
 			copyCodeBtn.classList.add('content-copy-btn');
 
-			if (addCopyBtn && window.hasOwnProperty('SyntaxHighlighter'))
-				copyCodeBtn.addEventListener('click', copyToClipboard);
+			if (!!copyToClipboard && window.hasOwnProperty('SyntaxHighlighter'))

```

Comment:
There is a lot of magic involved with that `copyToClipboard` function. If there is a function, is should be invoked, when there is null, don't. Wouldn't it be simpler and more clear to extract `var isClipboardSupported` to `isClipboardSupported` function and check here if it evalues to true? I would rather have a `copyToClipboard` function that doesn't need to be initialized in some magic:
```
if (isClipboardSupported()) {
    // (...)
    copyToClipboard(...);
}
```